com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -175,22 +175,28 @@ public partial class UIWidgetsPanel : RawImage, IUIWidgetsWindow {
         }
 
         protected void AddFont(string family, TextFont font) {
-            _internalTextFonts[key: family] = font;
+            _configurations._internalTextFonts[key: family] = font;
         }
 
         protected void AddFont(string family, List<string> assets, List<int> weights) {
             if (assets.Count != weights.Count) {
                 Debug.LogError($"The size of {family}‘s assets should be equal to the weights'.");
                 return;
             }
+
             var textFont = new TextFont {family = family};
             var fonts = new Font[assets.Count];
             for (var j = 0; j < assets.Count; j++) {
                 var font = new Font {asset = assets[index: j], weight = weights[index: j]};
                 fonts[j] = font;
             }
+
             textFont.fonts = fonts;
-            AddFont(family, textFont);
+            AddFont(family: family, font: textFont);
+        }
+
+        protected void SetShowDebugLog(bool showDebugLog) {
+            _configurations._showDebugLog = showDebugLog || m_ShowDebugLog;
```

Comment:
change this method to DisableDebugLog()

the logics of configuring showDebugLog for UIWidgetsPanel is the same as textFonts:
in the OnEnable, we first call onEnable where the user can call DisableDebugLog to set the flag to false. Then we also check the showDebugLog checkbox in the inspector window, and if it is set as false, we call DisableDebugLog again.